src/org/omegat/util/Language.java

```diff
@@ -120,11 +123,7 @@ public String toString() {
      * Returns a string representation as an ISO language code (XX-YY).
      */
     public String getLanguage() {
-        String langstring = getLanguageCode();
-        if (!langstring.isEmpty() && !getCountryCode().isEmpty()) {
-            langstring += "-" + getCountryCode();
-        }
-        return langstring;
+        return this.locale.toLanguageTag();
```

Comment:
Personally I wouldn't bother with `this` to refer to instance fields unless they were being shadowed by a local variable. But if you like it you can leave it.

Reply:
Fully agreed, I thought it was included in the Eclipse source cleanup config for the project -- I didn't bother to check.